tools/codegen/src/main.rs

```diff
@@ -673,7 +675,8 @@ impl GitHubTokens {
         }
     }
 }
-static GITHUB_TOKENS: LazyLock<GitHubTokens> = LazyLock::new(|| {
+// TODO: Use std::sync::LazyLock once 1.80 is released
+static GITHUB_TOKENS: Lazy<GitHubTokens> = Lazy::new(|| {
```

Comment:
I'd like to use OnceLock instead adding new dep (invert of https://github.com/taiki-e/cargo-config2/commit/614d68ef65566839aa7b5fec4e9b2eb5ba824e65).

Reply:
Sure, I will do it tomorrow (it's late here).

Reply:
Updated.